src/lib/dev-environment/dev-environment-core.js

```diff
@@ -73,6 +73,12 @@ type WordPressTag = {
 }
 
 export async function startEnvironment( slug: string, options: StartEnvironmentOptions ): Promise<void> {
+	const allEnvNames = getAllEnvironmentNames();
+	if ( allEnvNames.length > 1 ) {
```

Comment:
What do you think about this (I am not insisting on it):

```suggestion
	if ( allEnvNames.length > 1 && ! allEnvNames.includes( 'vip-local' ) ) {
```

The reason is that when I create a slugless environment, I do not expect to be forced to specify `--slug vip-local` parameter

Reply:
I think we should have to specify it if there's more than one environment...or else how would we know which one to target?

Reply:
Well, I mean I can have:

```bash
vip dev-env create --slug=test1
vip dev-env create --slug=test2
```

Those are smth like "permanent environments" where I work on something. Then, when I want to test something quickly, I sometimes do `vip dev-env create && vip-dev-env start`. This starts my "anonymous" (`vip-local`) environment.

Reply:
```
vip dev-env create --slug test1
vip dev-env start
```

The second command will fail if there is no "anonymous" environment